examples/mistral/readme.md

```diff
@@ -28,10 +28,15 @@ git config --system core.longpaths true
 ```
 
 ## Usage
+CPU:
 ```bash
 python -m olive.workflows.run --config mistral_optimize.json
 ```
 
+GPU:
+```bash
+python -m olive.workflows.run --config mistral_fp16_optimize.json
```

Comment:
do we need update the example doc https://microsoft.github.io/Olive/examples.html by adding the float16?

Reply:
I checked the doc and saw fp16 but turns out it was for phi2 and not mistral. 

Yeah, we should add list the cuda/fp16 setting.

Reply:
@apsonawane could you update `docs/source/examples.md` too?

Reply:
Done